extensions/websockets-next/server/runtime/src/main/java/io/quarkus/websockets/next/OnMessage.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkus.websockets.next;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import io.smallrye.common.annotation.Experimental;
+
+/**
+ * A method of an {@link WebSocket} endpoint annotated with this annotation is invoked when an incoming message is received.
```

Comment:
There can only be one `@OnMessage` method in the endpoint class, right? If so, I think this javadoc should say that. (IIUC, this also applies to `@OnOpen` and `@OnClose`.)